client/blocks/author-selector/index.jsx

```diff
@@ -95,7 +95,7 @@ const SwitcherShell = React.createClass( {
 					{ ( this.props.fetchOptions.search || users.length > 10 ) &&
 						<Search
 							onSearch={ this._onSearch }
-							placeholder={ this.translate( 'Find Author…', { context: 'search label' } ) }
+							placeholder={ this.translate( 'Find Author...', { context: 'search label' } ) }
```

Comment:
:wave: I've found a possible matching string that has already been translated **27** times:
`translate( 'Find Author…', { context: 'search label'} )` *ES Score: 8.99*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).